spec/requests/review_pages_spec.rb

```diff
@@ -329,7 +329,7 @@
       it "links to delete an invitation" do
         page.find(".fa-trash").click
         current_path.should == root_path
-        page.should have_selector('.flash-notice', text: 'bob@thoughtworks.com\'s invitation has been deleted.')
+        page.should have_selector('.flash-success', text: 'bob@thoughtworks.com\'s invitation has been deleted.')
```

Comment:
Line is too long. [113/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.